### PR TITLE
Added recent new content to Saltern & renovated Medical

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
I redesigned medical to be a bit more resilient during atmos emergencies, and I added a lot of the new content from the past few minutes / months (rollerbeds, crew monitoring console, cremator)
**Screenshots**

salternmedupper
salternmedlower
medstorage

**Changelog**

:cl: Rane
- add: Saltern now has all the latest & greatest objects from the past 2 months.
- tweak: Saltern medical has been renovated.

### CODE REVIEW

**Comment by Elijahrane** [comment]
Hmmm, odd test failure, I don't think I touched those

**Comment by Elijahrane** [comment]
The test is failing because of my lab coat with stuff inside it, but it works fine in-game.
https://cdn.discordapp.com/attachments/560845886263918612/925636812670636062/unknown.png

**Comment by Elijahrane** [comment]
Now that the bug is logged I removed the joke causing it, should be ready to review IMO

**Comment by Efruit** [comment]
Why does medbay in particular need inflatable walls or tools for atmos issues? That's an engineering problem, not a medical one. Shouldn't the engineers be handling it instead of doctors?

**Comment by Elijahrane** [comment]
1. There are no tools besides the wrench and crowbar, which is standard for medical.
2. Engineering would just fix the problem rather than using inflatable walls.

**Comment by Elijahrane** [comment]
I'm going to add in wardrobes now, don't merge for a bit

**Comment by Elijahrane** [comment]
I moved the inflatable barricades from med storage to med maint like they are on packed, I think the main complaint is addressed. Ready I think

**Comment by metalgearsloth** [comment]
Any reason the rollerbeds aren't aligned?

**Comment by Elijahrane** [comment]
No, not really

File: Resources/Maps/saltern.yml [hunk 1]
```diff
@@ -75160,6 +75164,33 @@ entities:
...
+    parent: 853
+    type: Transform
+- uid: 7127
+  type: PonderingOrb
+  components:
+  - pos: -25.5,22.5
+    parent: 853
+    type: Transform
+  - canCollide: False
+    type: Physics
```
Comment (metalgearsloth) [inline Resources/Maps/saltern.yml:75186]: ?

Comment (Efruit) [reply Resources/Maps/saltern.yml:75186]: # 🔘️🤔️💭️

Comment (Elijahrane) [reply Resources/Maps/saltern.yml:75186]: It's well hidden

**Pull request merged**